src/nuxt.config.ts

```diff
@@ -3,6 +3,14 @@ import type { NuxtConfig } from '@nuxt/types';
 import { siJellyfin as jellyfinIcon } from 'simple-icons/icons';
 
 const config: NuxtConfig = {
+  /*
+   ** Nuxt Environment Variables
+   ** See https://nuxtjs.org/docs/configuration-glossary/configuration-env
+   */
+  env: {
+    // Comma separated list of predefined servers (I.e. 'http://192.168.1.1' or 'http://192.168.1.1, https://your.server.domain')
+    server_url_list: process.env.DEFAULT_SERVERS || ''
```

Comment:
I guess the OR could be omitted, that'd only make the `server_url_list` undefined I guess?

Reply:
Actually env is of type Record <string, string> if you go to see the NuxtOptionsEnv interface :)

Only for this

Reply:
No don't worry, I haven't done my homework, you're right ^^ Then maybe default to `undefined` instead of empty string?

Reply:
But will have the same problem bc is not a string no? :)

I pick '' cause it's a string and returns true in if (!value) condition (which is the one i use in the parse function to fallback to [])

Reply:
Ok I'm stupid. Sorry ^^